corehq/apps/registry/helper.py

```diff
@@ -20,6 +21,10 @@ def __init__(self, current_domain, registry_slug=None, registry=None):
             self.registry_slug = registry_slug
             self._registry = None
 
+    def check_access(self, couch_user):
```

Comment:
Should this also check the feature flag?

Reply:
No, this code is only called after the FF is already checked.